packages/console/src/components/Topbar/TenantSelector/TenantDropdownItem/index.tsx

```diff
@@ -44,7 +47,12 @@ function TenantDropdownItem({ tenantData, isSelected, onClick }: Props) {
           />
         </div>
         <div className={styles.planName}>
-          <PlanName name={tenantPlan.name} />
+          {/* Todo: @xiaoyijun remove dev tenant feature switch */}
+          {isDevFeaturesEnabled && tag === TenantTag.Development ? (
+            <DynamicT forKey="subscription.no_subscription" />
+          ) : (
+            <PlanName name={tenantPlan.name} />
+          )}
```

Comment:
Can we write the logic in `<PlanName />` component? So that whenever the tenantPlan is empty, we fallback to the `No subscription` content